### DISCUSSION TITLE
How to open multiple projects without using cli?

### DISCUSSION THREAD

**Discussion by failable** [zed-industries/zed#8611] [Feature Requests]
When using cli `zed project1`, `zed project2` can open multiple projects in different windows at the same time. 

When using `"projects::OpenRecent"`, opening new project will replace the current one. Switching back to the "replaced" project will restart the LSP server. That's a bit annoying especially for LSP servers with slow startup like Rust Analyzer.

How to open multiple projects without using cli? Thanks.

**Comment by Moshyfawn** [comment]
Does `workspace: open` from the command palette does what you're looking for?

**Reply by failable** [reply]
Hi, I'm afraid it does not. Because it requires to using finder to "cd" in the project dir. That is something like I am trying to avoid when using cli, i.e. first "cd" into the path in shell, then`zed`. I wish I can switch projects easily by using simple keystrokes when working with multiple projects at the same time. e.g. using the same keystrokes to open or switch project (window), if the project is not open yet, just open and switch to it.

**Reply by Moshyfawn** [reply]
Ah, I see what you're after.

I think what you want is `cmd-alt-o` to open the recent projects drop down, then select the project you want and hold `cmd` as you open it. This should open your project in a new window.

**Reply by failable** [reply]
Thanks, that works!

**Reply by hasan-furkan** [reply]
hello. how can i open more than one project in the header section on a single screen. as an example i am leaving vscode screenshot 

<img width="493" alt="image" src="https://github.com/user-attachments/assets/31d04958-1d9a-49fd-b3b9-ecb6cb82fac7">

**Reply by nmassardot** [reply]
@hasan-furkan i have the same question, did you find out how to do it?

**Reply by Azir-11** [reply]
> hello. how can i open more than one project in the header section on a single screen. as an example i am leaving vscode screenshot
> 

You can change use_stystem_cindow_tabs to true in the settings, and then create a new window in full screen mode, which will present the form you need

<img width="2213" height="110" alt="image" src="https://github.com/user-attachments/assets/5cf07086-1723-4e23-9cc3-a5e88355000b" />

**Comment by ghenry** [comment]
On Linux I can do this holding Ctrl when clicking to project to open in an existing project. Thanks! Saved me losing things. Works like this by default on macOS though. Is there a setting for this on Linux? Open all projects in a new window?
